apps/ui/src/views/User.vue

```diff
@@ -38,6 +46,21 @@ const socials = computed(() => getSocialNetworksLink(user.value));
 
 const cb = computed(() => getCacheHash(user.value?.avatar));
 
+async function loadUserMetadata(userId: string) {
+  userMetadata.loading = true;
+
+  try {
+    const userStats = await getUserStats(userId);
+
+    userMetadata.followers_count = userStats.followers_count;
+    userMetadata.following_count = userStats.following_count;
+    userMetadata.loading = false;
+    userMetadata.loaded = true;
+  } catch (e) {
+    userMetadata.loading = false;
```

Comment:
maybe we should log error to understand what is going wrong, just in case
```suggestion
    console.log(e);
    userMetadata.loading = false;
```